smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -419,9 +419,9 @@ contract BridgeBank is BankStorage, CosmosBank, EthereumWhiteList, CosmosWhiteLi
   }
 
   /**
-   * @dev Fetches de decimals of a token by address
+   * @dev Fetches the decimals of a token by address
    * @param token The bridgeTokens's address
-   * @return The bridgeTokens's decimals or 18
+   * @return The bridgeTokens's decimals or 0
```

Comment:
Do we need getDecimals at all?

Reply:
That structure of events expects them and a lot of the architecture already written expects them. Whether the current system needs them or not is something that could be evaluated, I assume not, but at this point, I would rather keep it in than rearchitect this area for MVP. I am trying to make the contract changes be as minimally invasive as possible.

Reply:
I meant the function itself - just call decimals on the contract instead of the function, as a way to reduce contract size.  Looks like we don't need it though.